e2e/artifact/download.go

```diff
@@ -94,37 +96,37 @@ func (d *Downloader) downloadPublishingInfo(ctx context.Context, tempDir, artifa
 	zipOut, err := os.Create(zipName)
 	if err != nil {
 		d.errChan <- fmt.Errorf("unable to create zip archive %s: %w", zipName, err)
-		return
+		panic(err)
```

Comment:
won't those panic prevent other test cases from the 2e2 suite to complete ?  Do we know which binary will exit from this panic call ?

If each testcases are separate coroutine and the suite will continue, this is fine

Reply:
This code path is only run when you run the e2e tests locally using a custom build. It will stop your tests running if the VHD for one of the tests doesn't exist. Because if the tests run with the wrong VHD, then you spend half a day trying to debug your tests on the wrong VHD. Which is super painful.